Refactoring des 3 phrases

GuillaumeAV opened the pull request:
Issu d'un brainstorming très stimulant de 45 mn avec le cher @ReX-AV-Gab ;-)

srosset81 committed:
Revert back Pierre's work

GuillaumeAV committed:
Refactoring des 3 phrases

Issu d'un brainstorming très stimulant de 45 mn avec le cher @ReX-AV-Gab ;-)

bouviermullerp reviewed (commented):
Ha vindiou, ça m’embête de venir m'immiscer après un branstorming de 45 minutes !
Néanmoins, je me permets :
_Standard & interoperable
Based on semantic web standards & the SOLID specification, Semapps gives the web its full potential._
-> je pense qu'on pourrait nous reprocher de faire l'amalgame entre interopérable et sémantique qui sont 2 choses différentes (il me semble, d'après ce que j'ai compris).
Tu peux avoir une base de données interopérable sans qu'elle soit sémantique et inversement. Je pense que ça peut engendrer un amalgame.
Je laisse aussi exprimer mon égo (et peut être qu'il n'a pas sa place), _Semapps gives the web its full potential_ je trouve que c'est un peu ambitieux? . SemApps is built on the roots of the web par exemple. Mais peut être que ça fait moins rêver et qu'il faut être très ambitieux pour faire de belles choses, de viser la lune. Alors ce n'est pas une objection - juste un point de vue non bloquant, avec l'envie d'apporter une perspective.

Proposition de reformulation : Decentralized & federated
Allowing multiple platforms to be both self-**contained** -> **hosted** and linked together, Semapps gives you back control.

Modular
Built on a micro-services architecture, SemApps ensures a high level of flexibility, **sobriety** and scalability.

GuillaumeAV committed:
update according to Peter & Guillaume's feedbacks ;-)

is built on the roots of the web [...]
[...] and allows you to leverage network effects
[...] is fully customizable, it ensures a high level of flexibility, sobriety and [...]

GuillaumeAV committed:
update première phrase ... and brings out

update première phrase ... and brings out

GuillaumeAV commented:
@ReX-AV-Gab @bouviermullerp @srosset81 @simonLouvet 
J'ai un poil retravaillé les 3 phrases à la lumière du commentaire de Pierre et de mes propres pérégrinations ... Est-ce que ca vous va comme ça ? 
N'hésitez pas à reviewer ;-)
